core/tauri-runtime-wry/src/lib.rs

```diff
@@ -1578,6 +1579,14 @@ impl<T: UserEvent> WindowDispatch<T> for WryWindowDispatcher<T> {
     Ok(window_getter!(self, WindowMessage::PrimaryMonitor)?.map(|m| MonitorHandleWrapper(m).into()))
   }
 
+  fn monitor_from_point(&self, x: f64, y: f64) -> Result<Option<Monitor>> {
+    // TODO: how do I pass x and y?
+    Ok(
+      window_getter!(self, WindowMessage::MonitorFromPoint)?
+        .map(|m| MonitorHandleWrapper(m).into()),
+    )
+  }
```

Comment:
I think I'll need to pass on x and y to the WindowMessage but I'm not sure how? Adding it as arguments just gives me this error 🤔
<img width="1008" alt="Screenshot 2024-05-14 at 7 16 47 AM" src="https://github.com/tauri-apps/tauri/assets/30227512/5c5d59be-a95f-4f76-8a7f-c0a3fdc50300">

Reply:
It expects the Sender<...> type as first argument, but by default seems to get it even when no arguments are passed, so how come it demands that to be explicitly passed when I give additionally give it x and y? 😂

Reply:
you will have to use a manual approach here 

```rs
 	let (tx, rx) = channel();
 	send_user_message(
      &self.context,
      Message::Window(
        self.window_id,
        WindowMessage::MonitorFromPoint(tx, x, y),
      ),
    )
    
    rx
      .recv()
      .map_err(|_| crate::Error::FailedToReceiveMessage)
```

Reply:
Thanks, this helped! I think I've approached it correctly now.